app/assets/stylesheets/modules/gem.css

```diff
@@ -31,6 +31,10 @@
       overflow: auto;
       border-bottom: 1px solid #c1c4ca; } }
 
+.owners {
```

Comment:
I would argue that yes it is :) `.owners` is a block and `.gem__owners` are its elements. Possibly it could be better named. I am open to suggestions :email: